packages/block-library/src/blocks/header/init.php

```diff
@@ -90,6 +90,10 @@ class="<?php echo esc_attr( join( ' ', $classes ) ); ?>"
 						?>
 					</div>
 				</div>
+
+				<?php if ($header_is_simple ) {
+					echo renderReadingBar();
```

Comment:
We should probably keep camelCase for JS and try to stick to underscores in php files